src/style.css

```
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star :not(:last-child) {
```

```suggestion
.stars__star:not(:last-child) {
```
it does matter :)
with space, it applies to nested in `stars__star` element